src/libs/DistanceRequestUtils.ts

```diff
@@ -43,11 +43,11 @@ const METERS_TO_MILES = 0.000621371; // There are approximately 0.000621371 mile
  * Retrieves the mileage rates for given policy.
  *
  * @param policy - The policy from which to extract the mileage rates.
- * @param includeDisableRate - Should include disable rate
+ * @param includeDisabledRates - Should include disable rate
```

Comment:
Please update the description to `Whether disabled rates should be included`.

Or we could remove the description for the params and return. Actually that would be best since it says it returns an array which is wrong.